tests/product.test.js

```diff
@@ -5,12 +5,18 @@ const { models, connectDb } = require ('../dal/database');
 // dummy documents
 const products = [{
   _id: new ObjectID(),
-  product: 'mobile phone',
-  manufacturers: ["Samsung", "AT&T", "LG Electronics"]
+  barcodeId: "8152210266539",
+  name: "Macbook Pro 2018",
+  category: "laptop",
+  manufacturer: "Apple",
+  ESG: "1"
 }, {
   _id: new ObjectID(),
-  product: 'smartphone',
-  manufacturers: ["Nokia", "Lenovo", "Oppo"]
+  barcodeId: "07350053850019",
+  name: "Microsoft Surface Pro 6",
+  category: "laptop",
+  manufacturer: "Microsoft",
+  ESG: "0.8"
```

Comment:
Same here

Reply:
These are dummy documents for unit tests. Should not affect the deployment.